src/Resources/views/CRUD/Association/edit_many_to_many.html.twig

```diff
@@ -64,10 +64,7 @@ file that was distributed with this source code.
                         {% for nested_group_field_name, nested_group_field in form.children %}
                             {% for field_name, nested_field in nested_group_field.children %}
                                 {% if sonata_admin.field_description.associationadmin.hasformfielddescriptions(field_name) is defined %}
-                                    {{ form_row(nested_field, {
-                                        'inline': 'natural',
-                                        'edit'  : 'inline'
-                                    }) }}
+                                    {{ form_row(nested_field, {'edit': 'inline'}) }}
```

Comment:
This will not break other things? Can you try to figure why we use the `natural` value?

Reply:
When I tried, this worked well and didn't change the rendering of other field of the CollectionType. But I don't know enough this code/these options to be sure that this was not here for a reason.

I made some more investigations.
For what I understood, both inline and edit value are not needed. The form render is not realized by sonata but the FormType::class used. So these value seems to be never used because no FormType has a edit option or an inline option.
It was just used by CKeditor, by chance, since CKeditor has an inline options.